entries/async.xml

```diff
@@ -2,11 +2,16 @@
 <?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
 <entry type="method" name="async">
   <title>async()</title>
+  <signature>
+    <argument name="acceptCallCount" type="Number" optional="true" default="1">
+      <desc>Number accept a call count of callback of assert.async().</desc>
+    </argument>
+  </signature>
   <desc>
     Instruct QUnit to wait for an asynchronous operation.
   </desc>
   <longdesc>
-    <p>When your test has any asynchronous exit points, call <code>assert.async()</code> to get a unique resolution callback for each async operation. The callback returned from <code>assert.async()</code> will throw an Error if is invoked more than once.</p>
+    <p>When your test has any asynchronous exit points, call <code>assert.async()</code> to get a unique resolution callback for each async operation. The callback returned from <code>assert.async()</code> will throw an Error if is invoked more than accept call count.</p>
```

Comment:
How about "The callback returned from <code>assert.async()</code> will throw an Error if it is invoked more than once (or more often than the accepted call count, if provided)."